packages/chains/src/chains/cosmos.ts

```diff
@@ -0,0 +1,32 @@
+import { Chain, Extensions } from "@anoma/types";
+
+const DEFAULT_ALIAS = "Cosmos Testnet";
+const DEFAULT_CHAIN_ID = "cosmos-75a7e12.69483d59a9fb174";
```

Comment:
Do we have any way to override these conveniently?

Reply:
But likely that is not MVP

Reply:
We can provide overrides in the `.env` file, which needs to be there when we want to point to a local testnet (for example) - these get loaded from `process.env` just below: